.github/workflows/dev-deploy-ws.yml

```diff
@@ -55,11 +55,6 @@ jobs:
               }
             }
 
-      - name: Setup QEMU
-        uses: docker/setup-qemu-action@v3
-        with:
-          platforms: linux/amd64,linux/arm64
-
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Currently we are using QEMU in 13 files in the repo, this PR removes 3 of them. The question is:
1. What purpose does QEMU server?
2. Can we remove it in those other places too?
‏

Reply:
It's a relic from the past when we were doing multi architecture deployments.